vars/notifyBuildResult.groovy

```diff
@@ -69,6 +69,8 @@ def call(Map args = [:]) {
       def aggregateComments = args.get('aggregateComments', true)
       def flakyDisableGHIssueCreation = args.get('flakyDisableGHIssueCreation', false)
       catchError(message: 'There were some failures with the notifications', buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
+        // FIXME: EnvironmentContributions are not populated in the env global map
+        setEnvVar('OTEL_ELASTIC_URL', sh(label: 'OTEL_ELASTIC_URL', script: 'echo -n "${OTEL_ELASTIC_URL}"', returnStdout: true))
```

Comment:
EnvironmentContributions are only available in the shell context rather than the global `env` variable.

Reply:
As soon as https://github.com/jenkinsci/opentelemetry-plugin/pull/71 gets merged and released then we won't need this fix

Reply:
```suggestion
```